File: App_005/app/src/main/java/uta/fisei/app_005/MainActivity.java
package uta.fisei.app_005;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    private EditText editTextUser;
    private EditText editTextPassword;
    private Button buttonOk;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        editTextUser = findViewById(R.id.editTextUser);
        editTextPassword = findViewById(R.id.editTextPassword);
        buttonOk = findViewById(R.id.buttonOk);
    }

    public void onClickButtonOk(View view) {
        String user = editTextUser.getText().toString();
        String password = editTextPassword.getText().toString();
        if (!user.matches("") && !password.matches("")){
            Intent intent = new Intent(this, DataBaseActivity.class);
            startActivity(intent);
        }else {
            Toast.makeText(this, "Debe Ingresar los datos", Toast.LENGTH_SHORT).show();
        }
    }
}

File: Kotlin_Ejercicios/Kotlin_app_002/app/src/main/java/uta/fisei/kotlin_app_002/MainActivity.kt
package uta.fisei.kotlin_app_002

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.Toast
import uta.fisei.kotlin_app_002.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        //setContentView(R.layout.activity_main)

        val binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        /*val editTextUserName = findViewById<EditText>(R.id.editTextUserName)
        val editTextPassword = findViewById<EditText>(R.id.editTextPassword)
        val buttonOk = findViewById<Button>(R.id.buttonOk)

        buttonOk.setOnClickListener {
            val username = editTextUserName.text.toString()
            val password = editTextPassword.text.toString()

            if (username != "" && password != "") {
                val intent = Intent(this, SecondActivity::class.java)
                startActivity(intent)
            } else {
                Toast.makeText(this, "Los datos son Obligatorios", Toast.LENGTH_LONG).show()
            }
        }*/

        binding.buttonOk.setOnClickListener {
            val username = binding.editTextUserName.text.toString()
            val password = binding.editTextPassword.text.toString()
            if (username != "" && password != "") {
                val intent = Intent(this, SecondActivity::class.java)
                startActivity(intent)
            } else {
                Toast.makeText(this, "Los datos son Requeridos", Toast.LENGTH_LONG).show()
            }
        }
    }
}
